README.md

```diff
@@ -1,69 +1,54 @@
 # Kosha Discord Connector
 
-Discord is a VoIP and instant messaging social platform. Users have the ability to communicate with voice calls, video calls, text messaging, media and files in private chats or as part of communities called "servers".
+Discord is a social platform that enables you to communicate via voice, video, or chat—and to share files and media—in public or private communities called "guilds" (or "servers").
 
-The connector APIs allow you to perform 'RESTful' operations such as reading, modifying, adding or deleting data from your helpdesk. The APIs also support Cross-Origin Resource Sharing (CORS).
+Using the Kosha Discord connector, you can perform REST API operations such as reading, modifying, adding, or deleting data from your Discord servers. The connector also supports Cross-Origin Resource Sharing (CORS).
```

Comment:
Do we need the sentence about CORS here?

Reply:
We probably don't need that here.